doc/sphinx/changes.rst

```diff
@@ -14,6 +14,100 @@ Version 8.14
+
+  - The :ref:`internal representation <814CaseRepresentation>` of `match` has changed
+    to a more space-efficient and cleaner structure, allowing to fix a completeness
+    issue in presence of cumulative inductive types in the type-checker.
+    The internal representation is now closer to the user-level view of `match`,
+    where the argument context of branches and the inductive binders `in` and `as`
+    do not carry type annotations.
+
+  - A :ref:`new <814CoqNative>` `coqnative` binary performs separate native compilation
+    of libraries, starting from a `vo` file, and is supported by `coq_makefile`.
+    :ref:`Linking of native code libraries <814NativeLinking>` is now performed lazily,
+    bringing a gain of performance on some systems.
+
+  - :ref:`Improvements <814TCCanon>` of typeclasses and canonical structure
+    resolution, allowing more terms to be considered as classes or keys.
+
+  - More control of :ref:`notations <814Notations>` declarations and support
+    for primitive types in string and numeral notations.
+
+  - :ref:`Removal <814Tactics>` of deprecated tactics, notably `omega`, replaced by
+    a greatly improved `lia`, along with many bugfixes.
+
+  - New APIs for interaction with Ltac1, manipulation of inductive types and
+    a Printf module in :ref:`Ltac2 <814Ltac2>`.
+
+  - CoqIDE now integrate an :ref:`Ltac debugger <814CoqIDE>`.
+
+  - Many :ref:`changes and additions <814Stdlib>` to the standard library in the numbers,
+    vectors and lists libraries. A new signed primitive integers library `Sint63`
+    is available in addition to the unsigned `Uint63` library.
+
+  - Coq has been :ref:`split <814Dune>` into `coq-core` and `coq-stdlib` packages
+    that are now built using `dune`.
+
+See the `Changes in 8.14.0`_ section below for the detailed list of changes,
+including potentially breaking changes marked with **Changed**.
+Coq's `reference manual <https://coq.github.io/doc/v8.14/refman>`_,
+`documentation of the standard library <https://coq.github.io/doc/v8.14/stdlib>`_
+and `developer documentation of the ML API <https://coq.github.io/doc/v8.14/api>`_
+are also available.
+
+Maxime Dénès, Emilio Jesús Gallego Arias, Gaëtan Gilbert, Michael
+Soegtrop and Théo Zimmermann worked on maintaining and improving the
+continuous integration system and package building infrastructure.
+
+Erik Martin-Dorel has maintained the `Coq Docker images
+<https://hub.docker.com/r/coqorg/coq>`_ that are used in many Coq
+projects for continuous integration.
+
+The OPAM repository for Coq packages has been maintained by
+Guillaume Claret, Karl Palmskog, Matthieu Sozeau and Enrico Tassi with
+contributions from many users. A list of packages is available at
+https://coq.inria.fr/opam/www/.
+
+The `Coq Platform <https://github.com/coq/platform>`_ has been maintained
+by Enrico Tassi and Michael Soegtrop. TODO: It will soon have a release for Coq 8.14?
+
+Our current 32 maintainers are Yves Bertot, Frédéric Besson, Tej
```

Comment:
It seems that this list was also copied verbatim from the last release. We are now up to 34 maintainers. If I'm not mistaken this list is missing @Alizter and @proux01 and should otherwise be accurate. BTW, the Coq Team page should probably be updated as well.

Reply:
Thanks